book/website/community-handbook/bookdash/bookdash-application.md

```diff
@@ -0,0 +1,95 @@
+
+All these responses are securely handled under the [GDPR (General Data Protection Regulation) law](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) (learn about creating [GDPR-proof surveys](https://www.surveylegend.com/gdpr/how-to-make-gdpr-proof-surveys-or-forms/)).
+
+## Reviewing process
+
+Applications for _The Turing Way_ book dashes are reviewed by at least 2 members of _The Turing Way_ core team.
+A response form created by the organising members using the rubrics described below, the response from which are shared among all the reviewers during a designated meeting between the panel members.
+Scoring for each criterion is done between 0 to 2, where 2=criteria fully met, 1=criteria partially met, and 0=criteria not met.
+This rubric aims to evaluate the application across multiple aspects and avoid any personal bias panel members may have.
+
+*This rubric is adapted from the [Mozilla Open Leadership](https://foundation.mozilla.org/en/initiatives/mozilla-open-leaders/) and the [Open Life Science](https://openlifesci.org/) programs*
+
+(ch-bookdash-application-rubrics)=
+### Rubrics for scoring applications
+
+The invited reviewers will rate the participants across multiple dimensions with scores of 1, 2, or 3.
+
+* Has the applicant answered the application questions?
+* Does the applicant have a clear, feasible, and relevant idea of how they will contribute to the book dash?	This scoring corresponds generally to reviews of "not ready", "enthusiastic" and "clear" respectively:
+* Is the applicant enthusiastic about reproducibility and the aims of _The Turing Way_?
+1. Applicants who score **mostly 1s** do not have a clear idea/topic ready for contribution or do not have a clear understanding of the overall goals of _The Turing Way_ and book dash in general.
+2. Applicants who score **mostly 2s** are enthusiastic if not wholly suitable for the project/event, for example they may come with some ideas for a contribution that is currently not in the scope of the project/event.
+3. Applicants who score **mostly 3s** are clearly ready, goal-oriented, interested in contributing to the project, and excited to learn from others at the event.
+
+Reviewers are also asked whether they would recommend an applicant. The available answers are "No", "Yes", "Unsure" or "Wild-Card/Red-Flag (request to discuss)".
+Candidates who have ratings of "No" or "Unsure" across all reviewers will be rejected.
+All other candidates will be discussed at the panel selection.
+
+We ask all reviewers to finish the review with a few summary sentences to describe the application and the motivations for their scores to facilitate conversation during the selection panel discussion.
+Finally, reviewers will also disclose if they have any conflict of interest that may influence their review leading to any bias in their decision.
+
+Here are the main areas that we will look into when reviewing an application:
```

Comment:
Think these two sections (above and below this comment) duplicate so I would delete the one above and incorporate some of the contextual comments (i.e. people who are rejected by all reviewers are rejected outright) into the section below. The section below offers the most detail on the process through calling out the specific aspects reviewers are asked to rate people on.

Reply:
I need to think about this part. The part above this comment is in the introduction guide for the reviewers to explain the rationale of using this scoring system. The part below explains these scores for each question.

Maybe I need to explain that clearly to avoid the feeling of repeatition.

Reply:
Created a table to describe the criteria and avoid redundancy: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/32c2637bfe375cde0ec34669d78df6794e17a9ee